api/dss.yaml

```diff
@@ -780,6 +853,9 @@ components:
       pattern: ^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-4[0-9a-fA-F]{3}-[89abAB][0-9a-fA-F]{3}-[0-9a-fA-F]{12}$
       type: string
       example: 03e5572a-f733-49af-bc14-8a18bd53ee39
+  Version:
```

Comment:
YAML syntactical correctness depends on accuracy of indents; this indent is not correct and causes validation failure.